### PR TITLE
Updating URLs and benefits information

### PR DESCRIPTION
**Pull request by karlitoz007**
Updating URLs and benefits Information.

### CODE REVIEW

**Comment by xavierdecoster** [comment]
Can you keep the URLs as HTTPS please (where applicable)?

File: index.html [hunk 1]
```diff
@@ -18,17 +18,17 @@
 
 - perk: {company: "JetBrains", uri: "http://www.jetbrains.com/devnet/sponsorship/mvps/"}
   description: "We provide Microsoft Most Valuable Professionals with .NET tools and WebStorm licenses."
-  products: ["ReSharper", "dotTrace", "dotCover", "WebStorm"]
+  products: ["ReSharper", "dotTrace", "dotMemory", "dotCover", "WebStorm"]
 
-- perk: {company: "MyGet", uri: "https://www.myget.org/mvp"}
+- perk: {company: "MyGet", uri: "http://www.myget.org/mvp"}
```
Comment (xavierdecoster) [inline index.html]: Why this change to non-HTTPS? HTTPS is preferred

File: index.html [hunk 2]
```diff
@@ -18,17 +18,17 @@
...
   description: "MyGet allows you to create and host your own NuGet feed. Include packages from the official NuGet feed or upload your own NuGet packages. Compatible with NuGet, the Visual Studio Package Manager Console, Orchard, SymbolSource and many more."
   products: ["MyGet MVP Subscription"]
 
-- perk: {company: "Cerebrata", uri: "https://www.cerebrata.com/store/discounts"}
+- perk: {company: "Cerebrata", uri: "http://www.cerebrata.com/store/discounts"}
```
Comment (xavierdecoster) [inline index.html]: why no more HTTPS?

File: index.html [hunk 3]
```diff
@@ -84,9 +84,13 @@
   description: "SiaqoDB provides MVPs with a free Bundle license for their NoSQL embedded object and document database engine."
   products: ["SiaqoDB", "NoSQL"]
 
-- perk: {company: "elmah.io", uri: "https://elmah.io/"}
+- perk: {company: "elmah.io", uri: "http://elmah.io/"}
```
Comment (xavierdecoster) [inline index.html]: why no more HTTPS?

**Comment by xavierdecoster** [comment]
Good changes, I'll ship it when comments addressed :) Thanks for the contribution!

**Comment by karlitoz007** [comment]
The PR is now updated with a new commit. Thanks!

**Comment by xavierdecoster** [comment]
Thank you!
